src/scripts/main.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const mainImage = document.querySelector('#largeImg');
+
+  const thumbnails = document.querySelector('#thumbs');
+
+  function changeImage(e) {
+    const necessaryLink = e.target.closest('a');
+
+    if (necessaryLink) {
+      e.preventDefault();
+
+      const clickedImage = necessaryLink.querySelector('img');
```

Comment:
The variable name 'clickedImage' could be more descriptive. Consider renaming it to something like 'thumbnailImage'.